app/Http/Controllers/CustomFieldController.php

```
@@ -172,9 +179,30 @@ public function destroy($id)
     }
 
+    /**
+     * Sort fields order within a module
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function sortFields(Request $request)
+    {
+        $sortedValues = $request->sortedValues;
+        $module = $request->module; // Optional: if you want to sort within a specific module
```

PHPMD (2.15.0)

Avoid unused local variables such as '$module'. (undefined)

(UnusedLocalVariable)